dolly-backend-app/src/main/java/no/nav/dolly/service/BrukerService.java

```diff
@@ -116,13 +117,25 @@ public Bruker saveBrukerTilDB(Bruker b) {
 
     public int migrerBruker(Collection<String> navIdenter, String brukerId) {
         Optional<Bruker> bruker = brukerRepository.findBrukerByBrukerId(brukerId);
```

Comment:
I stedet for denne bør du kalle fetchOrCreateBruker() da vil bruker opprettes hvis den ikke fantes fra før.